archipelago/docs/en_Donkey Kong 64.md

```diff
@@ -10,22 +10,62 @@ Items which the player would normally acquire throughout the game have been move
 always able to be completed, but because of the item shuffle the player may need to access certain areas before they
 would in the vanilla game.
 
+In addition, the level entrances from DK Isles have been swapped around. This means that you may find Creepy Castle where the entrance for Jungle Japes normally is, and so on.
+
+Due to some current technical constraints brought on by the Archipelago framework, a number of changes have been made from the regular Donkey Kong 64 randomizer logic, including:
+- Tag Anywhere is considered in logic
+- Purchases from shopkeepers (Cranky/Funky/Candy) are free
+- B. Lockers costs are 0 Golden Bananas for all levels, except Hideout Helm which needs 60 Golden Bananas
+- Banana Medal checks are sent after collecting 40 of the same colored bananas in a world, down from 75
+
+These restrictions may be changed in future updates to this APWorld.
+
 ## What is the goal of Donkey Kong 64 when randomized?
 
-TBD
+Currently, two goals can be selected:
+
+- Defeat King K. Rool
+- Find all the Keys for K. Lumsy's cage
 
 ## What items and locations get shuffled?
 
-TBD
+### Items
+
+- Playable Kongs
+- Golden Bananas
+- Banana Fairies
+- Banana Medals
+- Battle Crowns
+- All Keys for K. Lumsy's cage, except Key 8
+- Blast-o-Matic blueprints
+- Instruments, guns and respective upgrades
```

Comment:
I'd simplify this and the "Cranky's Lab moves/potions" as just "Moves purchased from shops"